static/js/around-map.js

```diff
@@ -107,45 +111,47 @@ function urlFor (id) {
 }
 
 function findPoi (id) {
-  return pois.find(poi => poi._id == id)
+  return pois.find(poi => poi._id === id)
 }
 
 function highlight (poi_id) {
   const cardActive = document.querySelector('.current-card')
   const markerActive = document.querySelector('.current-marker')
-  const card = document.querySelector(`#card-${poi_id}`)
-  const marker = document.getElementById(poi_id)
   const superMarker = document.querySelector('.master-marker')
+  // querySelector don't works if poi_id begin with a number
+  const marker = document.getElementById(poi_id)
+  const card = document.querySelector(`#card-${poi_id}`)
   if(cardActive) {
-    cardActive.classList.toggle('current-card')
+    cardActive.classList.remove('current-card')
   }
   if(markerActive) {
-    markerActive.classList.toggle('current-marker')
+    markerActive.classList.remove('current-marker')
   }
   if(card) {
-    card.classList.toggle('current-card')
     marker.classList.toggle('current-marker')
+    card.classList.toggle('current-card')
```

Comment:
You just removed the `current-card` class from `cardActive`.
Are you trying to add it here? In which case you should explicitly use `classList.add(…`